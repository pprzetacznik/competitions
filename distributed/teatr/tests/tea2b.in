99999999 2 50 -1 5 8422264227670548395
