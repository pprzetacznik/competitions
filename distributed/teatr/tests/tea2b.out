2900011299055466
